source/configuration_manual/stats/index.rst

```diff
@@ -272,6 +272,15 @@ Metrics can be removed dynamically by running:
    doveadm stats remove <name>
 
 Examples:
+
+.. code-block:: none
+
+   doveadm stats add --description "Description of the metric" --exporter log-exporter --exporter-include "name timestamps" --fields "bytes_in bytes_out" --group-by "cmd_name tagged_reply_state" metric_name "event=imap_command_finished AND name=SELECT"
+
+.. code-block:: none
```

Comment:
Add also an example command to modify the just added metric in some way.

Reply:
What do you mean? metric can be added and removed, how to modify it?